config/jacoco.gradle

```diff
@@ -49,13 +47,9 @@ task jacocoTestReport(type: JacocoReport) {
         excludes: fileFilter
     )
 
-    sourceDirectories.from = files([
-        "$project.rootDir/app/src/main/java",
-        "$project.rootDir/data/src/main/java"])
+    sourceDirectories.from = files([android.sourceSets.main.java.srcDirs])
 
-    executionData.from = fileTree(dir: project.rootDir, includes: [
-        'app/build/jacoco/testStagingDebugUnitTest.exec',
-        'data/build/jacoco/testStagingDebugUnitTest.exec'])
```

Comment:
is this correct to remove `data` out of executionData?

Reply:
I'm not sure @luongvo but I think I will bring it back like above to make sure the file is exact.